manifests/04-deployment.yaml

```diff
@@ -24,16 +24,17 @@ spec:
         - "-configmap=cluster-monitoring-config"
         - "-logtostderr=true"
         - "-v=4"
-        #- "-tags=prometheus-operator=master"
-        #- "-tags=prometheus-config-reloader=master"
-        #- "-tags=config-reloader=master"
-        #- "-tags=prometheus=master"
-        #- "-tags=alertmanager=master"
-        #- "-tags=grafana=master"
-        #- "-tags=oauth-proxy=master"
-        #- "-tags=node-exporter=master"
-        #- "-tags=kube-state-metrics=master"
-        #- "-tags=kube-rbac-proxy=master"
+        - "-images=prometheus-operator=docker.io/openshift/origin-prometheus-operator:latest"
+        - "-images=prometheus-config-reloader=docker.io/openshift/origin-prometheus-config-reloader:latest"
+        - "-images=config-reloader=docker.io/openshift/origin-configmap-reload:latest"
+        - "-images=prometheus=docker.io/openshift/origin-prometheus:latest"
+        - "-images=alertmanager=docker.io/openshift/origin-prometheus-alertmanager:latest"
+        - "-images=grafana=docker.io/grafana/grafana:latest"
+        - "-images=oauth-proxy=docker.io/openshift/oauth-proxy:latest"
+        - "-images=node-exporter=docker.io/openshift/origin-prometheus-alertmanager:latest"
```

Comment:
Probably copy/paste error.  Should be openshift/origin-prometheus-node-exporter:latest.

Reply:
absolutely! updated :slightly_smiling_face: